core/schema/container.go

```diff
@@ -312,6 +313,36 @@ func (s *containerSchema) Install() {
 		dagql.Func("withExec", s.withExec).
 			View(AllVersion).
 			Doc(`Retrieves this container after executing the specified command inside it.`).
+			ArgDoc("args",
+				`Command to run instead of the container's default command (e.g., ["run", "main.go"]).`,
+				`If empty, the container's default command is used.`).
+			ArgDoc("useEntrypoint",
+				`If the container has an entrypoint, prepend it to the args.`).
+			ArgRemove("skipEntrypoint").
```

Comment:
This is a shorthand for removing this as a parameter - without this, you'd have to redefine the entire `args` struct, and have a new fun to take this.

Ideally, we'd have even more shorthands for common operations - since redeclaring all the common fields over and over is quite annoying.